deploy/deployChangedPackages.mjs

```diff
@@ -93,8 +102,10 @@ Assuming that this means we need to upload this package.`);
 
       uploaded.push(dirName);
     }
-  }
 
+    console.log("\n# Release notes:");
```

Comment:
I don't know the code well enough -- but it looks like this prints the release notes but doesn't upload them. Is that the intent?

Reply:
It does! I just figured it'd be nice to have that in the CI logs also